Zend/tests/closures/closure_get_current.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+Closure::getCurrent()
+--FILE--
+<?php
+
+$i = 1;
+
+$c = function ($p) use (&$i) {
+    $self = Closure::getCurrent();
+    var_dump($p, $i);
+    $i++;
+    if ($p < 10) {
+        $self($p + 1);
+    }
+};
+
+$c(1);
+var_dump($i);
+
```

Comment:
Closure in attribute should be tested as well.

Also probably a "fail" in class method and no function in global context.